assests/styleSheets/general.css

```diff
@@ -0,0 +1,16 @@
+@import "./reset.css";
+@import "./variables.css";
+
+body {
```

Comment:
The indentation for this file does not seem to be correct. All the other files follow two spaces but here are 4.